manifests/init.pp

```diff
@@ -184,11 +184,17 @@
       $pip_script = "${scripts}/pip"
     }
 
-    exec { 'pip-install':
-      command => "${easy_install} pip",
-      creates => $pip_script,
-      require => $setuptools_require,
-    }
+    if $::osfamily == 'windows' {
+        exec { 'pip-install':
+          command => "${easy_install} pip==20",
+          creates => $pip_script,
+          require => $setuptools_require,
+        } } else { exec { 'pip-install':
+          command => "${easy_install} pip",
+          creates => $pip_script,
+          require => $setuptools_require,
+        } }
+
```

Comment:
I would suggest we do this a bit differently.

1. Default a class variable called `easy_install_pip` that defaults to `pip`.
2. Make the `exec` block above use `command => "${easy_install} ${easy_install_pip}"`.

That way there's no change in behavior for this public package but we we can can the pip version via Hiera.